README.md

```diff
@@ -268,6 +268,57 @@ editors:
   - css_select: h1 > a
 ```
 
+#### `css_sort`
+
+Similar to [`css_select`](#css_select) but sorts the matched HTML elements before returning them.
```

Comment:
> matched HTML elements

This is surprising to me, as the tool is called "css_sort". If it acts on HTML, why isn't it called "html_sort"?

Reply:
Because, like css_select / remove it uses CSS selectors to do its job. It could be called `html_sort_by_css_selectors` which seems overly complicated. CSS is a common relation to HTML, so I think this is quite clear?

Reply:
Are you referring to [this](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_selectors)? I did not know about that! TIL!

Reply:
:ghost: 

https://github.com/EdJoPaTo/website-stalker/blob/4e6a5141775e0f265d344c94ea9316194abe47a2/README.md?plain=1#L258